src/test/java/PlaylistTests.java

```diff
@@ -1,13 +1,27 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
 
 import java.util.List;
 
-public class Homework19 extends BaseTest {
+public class PlaylistTests extends BaseTest{
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+        LoginPage lp = new LoginPage(getDriver());
+        lp.logIn();
+
+        HomePage hp = new HomePage(getDriver());
+        hp.addSongToPlaylist();
+
+        Assert.assertTrue(hp.isSongInPlaylist());
+    }
 
     @Test
     public void DeletePlaylistTest() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException and Thread. sleep`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.